packages/suite/src/utils/wallet/balanceUtils.ts

```diff
@@ -26,7 +28,7 @@ export const formatCoinBalance = (value: string) => {
             return fixedBalanceBig.toFixed(2);
         }
 
-        return fixedBalanceBig.toFixed();
+        return isTruncated ? `${fixedBalanceBig.toFixed()}...` : fixedBalanceBig.toFixed();
```

Comment:
Nitpicking a little but I would rather use the ellipsis character (… / U+2026) rather than the triple dot. Screen readers will read it as _ellipsis_ rather than _dot dot dot_ / _period period period_.

Reply:
good idea! It never hit me that there is special unicode char for ellipsis 🤦